src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -242,6 +242,7 @@ void GafferSceneUIModule::bindLightEditor()
 			)
 		) )
 		.def( "inspector", &InspectorColumn::inspector, return_value_policy<IECorePython::CastToIntrusivePtr>() )
+		.def( "cellData", &InspectorColumn::cellData, ( arg_( "path" ), arg_( "canceller" ) = object() ) )
```

Comment:
I think this is fixed in 81f350e0144954d201f9a3dede93e71e06cff1ad. When we discussed offline you mentioned one case concerning derived classes in Python that wouldn't get their `cellData` called. I'm still a bit fuzzy on that code path and how to handle it, but it also sounded like it may be unnecessary?